content/en/dashboards/scheduled_reports.md

```diff
@@ -18,97 +18,81 @@ aliases:
 
 ### Customize the report
 
-Finally, customize the report to provide recipients with more context or a tailored view. The optional description appears at the top of each report to provide more context on the dashboard.
+Finally, customize the report to provide recipients with more context or a tailored view. The optional description is included in the report email body.
 
-Click **Edit Template Variables** to modify the filters applied when the report is sent. These values do not affect the underlying dashboard's defaults. 
+Click **Edit Template Variables** to modify the filters applied when the report is sent. These values do not affect the underlying dashboard's defaults.
 
 To see the report before saving the schedule, click **Send Preview**. You can pause a report schedule at any time.
 
 ## Managing reports
-A single dashboard can have multiple scheduled reports with different settings, for example, to support different groups of stakeholders interested in the same dashboard. To see the reports on an existing dashboard, open the **Share** menu and select **Configure Reports**. 
-
-{{< img src="dashboards/scheduled_reports/configure_reports.png" alt="A view of the option to configure reports in the dashboard Share menu" style="width:50%;" >}}
+A single dashboard can have multiple scheduled reports with different settings, for example, to support different groups of stakeholders interested in the same dashboard. To see the reports on an existing dashboard, click the **Share** button and select **Configure Reports**.
 
 From the configuration modal that opens, you can pause an existing report or create a new report. To see and edit the details of an existing report, or delete the report, click **Edit**.
 
-{{< img src="dashboards/scheduled_reports/scheduled_reports_configuration_modal.png" alt="The configuration modal for scheduled reports, with two reports displayed, each showing their titles, tags, recipients, frequency, an option to toggle the report on or off, and a button to edit the report. At the bottom is a button to add a new report and a done button" style="width:100%;" >}}
+{{< img src="dashboards/scheduled_reports/scheduled_reports_configuration_modal.png" alt="The configuration modal for scheduled reports, with two reports displayed, each showing their titles, tags, recipients, frequency, an option to toggle the report on or off, and a button to edit the report. At the bottom is a button to add a new report and a done button" style="width:90%;" >}}
 
 ## Permissions
 
-Only users with the **Dashboard Report Write** permission can generate a report. This permission is turned on by default for Admins and off for all other roles.
+Users need the **Dashboards Report Write** [permission][permissions] to create and edit report schedules.
+This permission can be granted by another user with the **User Access Manage** permission and is available by default to users with the **Datadog Admin** [out-of-the-box role][ootb-role].
```

Comment:
I understand why you changed the format of the reference links, but I can't justify making an exception for this page without considering the 5000+ pages we manage 🙏 

```suggestion
This permission can be granted by another user with the **User Access Manage** permission, and is available by default to users with the **Datadog Admin** [out-of-the-box role][ootb-role].
```